subsys/net/lib/rest_client/Kconfig

```diff
@@ -10,24 +10,14 @@ menuconfig REST_CLIENT
 if REST_CLIENT
 
 config REST_CLIENT_REQUEST_TIMEOUT
-	int "Rest request timeout, in seconds"
+	int "REST request timeout, in seconds"
 	default 65
 	help
-	  Sets the timeout duration in seconds to wait for the response data.
-
-config REST_CLIENT_SCKT_SEND_TIMEOUT
-	int "Socket send timeout, in seconds"
-	default 60
-	help
-	  Sets the timeout duration in seconds to use when sending data.
-	  To disable, set the timeout duration to 0.
-
-config REST_CLIENT_SCKT_RECV_TIMEOUT
-	int "Socket receive timeout, in seconds"
-	default 60
-	help
-	  Sets the timeout duration in seconds to use when receiving data.
-	  To disable, set the timeout duration to 0.
+	  Sets the default timeout duration in seconds to wait for socket connection,
+	  sending the request and receiving the response data. The timeout is set
+	  individually for these different phases meaning REST request can take
+	  much longer than this given timeout. Technically timeout is set to
+	  socket send and receive timeout.
```

Comment:
Probably should also mention that timeouts can be disabled by setting this to 0.